mypy/fixup.py

```diff
@@ -0,0 +1,298 @@
+"""Fix up various things after deserialization."""
+
+from typing import Any, Dict, Optional, cast
+
+from mypy.nodes import (MypyFile, SymbolNode, SymbolTable, SymbolTableNode,
+                        TypeInfo, FuncDef, OverloadedFuncDef, Decorator, Var,
+                        TypeVarExpr, ClassDef,
+                        LDEF, MDEF, GDEF, MODULE_REF)
+from mypy.types import (CallableType, EllipsisType, Instance, Overloaded, TupleType,
+                        TypeList, TypeVarType, UnboundType, UnionType, TypeVisitor)
+from mypy.visitor import NodeVisitor
+
+
+def fixup_module_pass_one(tree: MypyFile, modules: Dict[str, MypyFile]) -> None:
+    node_fixer = NodeFixer(modules)
+    node_fixer.visit_symbol_table(tree.names)
+    # print('Done pass 1', tree.fullname())
```

Comment:
Commented out print statements. If these are useful, it's probably better to hind them behind a debug flag.